src/requests/index.ts

```diff
@@ -98,3 +105,56 @@ export class DomainClient {
     return formatRawProjectTypes(this.client, rawProjectTypes);
   }
 }
+
+// test set-up/tear-down utility
+export class TestClient {
+  private url: string;
+  private secret: string;
+
+  constructor({
```

Comment:
Could we throw an error here if `NODE_ENV !== "test"`? 

It's imperfect, but could act as somewhat of a guard against improper use and would make this more self-documenting at a minimum.

https://jestjs.io/docs/environment-variables#node_env